integrations/isaac_ros_3_2_yopo/isaac_ros_yopo_bringup/isaac_ros_yopo_bringup/runtime_health_monitor.py

```
@@ -0,0 +1,483 @@
+
+    @staticmethod
+    def _sequence_matches(
+        actual: Sequence[float],
+        expected: Tuple[float, ...],
+    ) -> bool:
+        return len(actual) == len(expected) and all(
+            isinstance(value, (int, float))
+            and not isinstance(value, bool)
+            and math.isfinite(float(value))
+            and float(value) == reference
+            for value, reference in zip(actual, expected)
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)